code/controllers/configuration.dm

```diff
@@ -243,7 +243,7 @@ var/list/gamemode_cache = list()
 	var/job_whitelist = FALSE //Do we use the job whitelist? Set to false so devs don't have to set config options.
 	var/panic_bunker = FALSE //is the panic bunker enabled?
 	var/panic_bunker_message = "Sorry! The panic bunker is enabled. Please head to our discord to get yourself added to the panic bunker bypass."
-	var/delist_population = 60 //What population do we automatically take ourselves off to hub at?
+	var/delist_population = 0 //What population do we automatically take ourselves off to hub at?
```

Comment:
Problem number 1: You can also edit the config to change this value instant of hardcode it.
Problem number 2: Fix `SSBabysit` actually properly finding all players instead of doing it like this, for consistency.